package.json

```diff
@@ -30,7 +30,10 @@
     "@semantic-release/github": "^7.2.0",
     "@semantic-release/release-notes-generator": "^9.0.1",
     "husky": "^4.2.5",
-    "semantic-release": "^17.0.7",
-    "patch-package": "^6.2.2"
+    "patch-package": "^6.2.2",
+    "semantic-release": "^17.0.7"
+  },
+  "dependencies": {
+    "@emulsify/cli": "^1.5.0"
```

Comment:
@callinmullaney This is probably a dev dependency.

Reply:
Agreed. Moved to the correct location.